diff --git a/dist/client/theme-default/components/VPHome.vue b/dist/client/theme-default/components/VPHome.vue
index d69a79b21a734adfc6253c74c32ed400fa25eb1f..a5aedb0855ad134bd3d91a18afc78dc2fd027aed 100644
--- a/dist/client/theme-default/components/VPHome.vue
+++ b/dist/client/theme-default/components/VPHome.vue
@@ -1,6 +1,7 @@
 <script setup lang="ts">
 import VPHomeHero from './VPHomeHero.vue'
 import VPHomeFeatures from './VPHomeFeatures.vue'
+import VPHomeContent from './VPHomeContent.vue'
 </script>
 
 <template>
@@ -19,7 +20,9 @@ import VPHomeFeatures from './VPHomeFeatures.vue'
     <VPHomeFeatures />
     <slot name="home-features-after" />
 
-    <Content />
+    <VPHomeContent>
+      <Content />
+    </VPHomeContent>
   </div>
 </template>
 
diff --git a/dist/client/theme-default/components/VPHomeContent.vue b/dist/client/theme-default/components/VPHomeContent.vue
new file mode 100644
index 0000000000000000000000000000000000000000..2b735857f88dd398b5eda0f25f65684199303b8a
--- /dev/null
+++ b/dist/client/theme-default/components/VPHomeContent.vue
@@ -0,0 +1,25 @@
+<template>
+  <div class="vp-doc container">
+    <slot />
+  </div>
+</template>
+
+<style scoped>
+.container {
+  margin: auto;
+  width: 100%;
+  max-width: 1280px;
+  padding: 0 24px;
+}
+@media (min-width: 640px) {
+  .container {
+    padding: 0 48px;
+  }
+}
+@media (min-width: 960px) {
+  .container {
+    width: 100%;
+    padding: 0 64px;
+  }
+}
+</style>